oe_theme.theme

```diff
@@ -557,18 +559,24 @@ function oe_theme_preprocess_links__language_block(array &$variables): void {
       ->setAbsolute(TRUE)
       ->toString();
 
-    // Get the language prefix.
-    $lang_prefix = \Drupal::configFactory()
-      ->get('language.negotiation')
-      ->get('url.prefixes.' . $language_code);
+    $eu_lang_list = Filters::getEuropeanUnionLanguageList();
+    $language_prefix = !empty($eu_lang_list[$language_code][2]) ? $eu_lang_list[$language_code][2] : $language_code;
```

Comment:
Maybe we can take out the static method `getEuropeanUnionLanguageList` from the filter and put it in its own class, so that we can add simple helper methods to get lang code etc. and hide all the implementation details in there. @voidtek is it ok if I do that?